ddtrace/internal/processor/trace.py

```diff
@@ -68,6 +68,25 @@ def process_trace(self, trace):
         return None
 
 
+@attr.s
+class TraceTopLevelSpanProcessor(TraceProcessor):
+    """Processor marks spans as top level"""
+
+    def process_trace(self, trace):
+        # type: (List[Span]) -> Optional[List[Span]]
+
+        if trace:
+            chunk_root = trace[0]
+            if chunk_root is chunk_root._local_root:
+                chunk_root.set_metric("_dd.top_level", 1)
```

Comment:
I wonder if we can skip this, since this is the scenario that is currently implied by the trace agent?

or better to set explicitly so there is less ambiguity?

Reply:
I think it's better to reduce ambiguity